applications/openshift/master/three_control_plane_nodes/rule.yml

```diff
@@ -0,0 +1,39 @@
+documentation_complete: true
+
+title: 'Ensure machine count of MachineConfigPool master is 3'
+
+description: |- 
+    To ensure, that the OpenShift control plane stays accessible on outage of a single master node, a 
+    number of 3 control plane nodes is required.
+
+rationale: |-
+    A highly-available OpenShift control plane requires 3 control plane nodes. This allows etcd to have
+    a functional quorum state, when a single control plane node is unavailable.
+
+identifiers: {}
+
+references:
+    bsi: APP.4.4.A19
+
+severity: medium
+
+ocil_clause: 'MachineConfigPool master has less than three nodes'
+
+ocil: |-
+    Run the following command to retrieve the count of nodes in the MachineConfigPool master.
+    <pre>$ oc get machineconfigpool master -o jsonpath='{.status.machineCount}'</pre>
+    Make sure that the number equals to 3.
+
+warnings:
+- general: |-
+    {{{ openshift_cluster_setting("/apis/machineconfiguration.openshift.io/v1/machineconfigpools/master") | indent(4) }}}
+
+template:
+  name: yamlfile_value
+  vars:
+    ocp_data: 'true'
+    filepath: /apis/machineconfiguration.openshift.io/v1/machineconfigpools/master
+    yamlpath: .status.machineCount
+    entity_check: at least one
+    values:
+    - value: '3'
```

Comment:
Is a setup with more than 3 control plane nodes supported and do we need to mark this as compliant?

Reply:
openshift is only supported with 3 master nodes or 1 master node (i.e. SNO).

In the case of a one master deployment, they wont have HA anyway, so they are incompliant (with good reasons). a tailored profile with rule exclusion should be good here, to have the user aware of this.